src/gui/Src/Gui/CPUDump.h

```diff
@@ -144,6 +146,7 @@ public slots:
         ViewFloatDouble,
         ViewFloatLongDouble,
         ViewAddress,
+        ViewAddressUnicode,

```

Comment:
I think it's better to add two new views `ViewAddressAscii` and `ViewAddressUnicode` in addition to the normal `ViewAddress`.

Also please add new enums to the end (because users of old versions will now get a different dump view when they open this updated x64dbg).